src/js/models/layer.js

```diff
@@ -504,6 +510,7 @@ define(function (require, exports, module) {
             proportionalScaling: false,
             isArtboard: !!isArtboard,
             bounds: isArtboard ? new Bounds(boundsDescriptor) : null,
+            maskbounds: isArtboard ? new Bounds(boundsDescriptor) : null,
```

Comment:
This line confuses me for a number of reasons actually. Why would `maskBounds` be the same as `bounds` here, but for artboards? It seems like the mask itself is being ignored.